README.md

```diff
@@ -1,56 +1,90 @@
 # Git Repository Basics
 
+
 ## Objectives
 
 1. Define Git and explain how it helps programmers control and manage changes
-2. Initialize a new Git repository with `git init`
-3. Check the state of your files using `git status`
-4. Stage files to commit using `git add` and `git commit`
-
-So, how do you use Git?
+2. State initializing Git repository with `git init`
+3. Use `git status` to check the status of your repository 
+4. Recognize how to keep track of files with `git add`
```

Comment:
I think this can be better. What does "keep track of" mean to someone learning `git` at this point? Something like "Add to revision control" seems more clear. Knowing what `git add` does, I see how "keep track of" is a synonym, but I don't think it's right for a beginner.